### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by ceady-tw** [Onemind-Services-LLC/react-icons-ng#541] [General]
What is your release schedule for this project?

My PR to add Material Round Icons was recently merged (thanks 🙏 ), and I would like to see a release for that to become available. Is that something I need to wait for, or can a manual release be triggered?

Thanks for this great library!

**Comment by abhi1693** [comment]
v5.2.2 is released. You should see it soon on the registry.

**Reply by ceady-tw** [reply]
Thanks! It looks like the publish failed?

https://github.com/Onemind-Services-LLC/react-icons-ng/actions/runs/10151363064
